Vester/Public/Invoke-Vester.ps1

```diff
@@ -16,6 +16,18 @@ function Invoke-Vester {
     Invoke-Vester then calls Pester to run each test file. The test files
     leverage PowerCLI to gather values for comparison/remediation.
 
+    .PARAMETER Script
```

Comment:
These aren't needed. The comments I've added directly above the actual parameters will auto-populate each `.PARAMETER` in the comment-based help.